scripts/sync-issues.sh

```
@@ -0,0 +1,101 @@
+fi
+
+echo "Creating issues in $repo..."
+
+create_issue() {
+  local title=$1 body=$2 milestone=$3 labels=$4
+  gh issue create \
+    --repo "$repo" \
+    --title "$title" \
+    --body "$body" \
+    --milestone "$milestone" \
+    $(for l in ${labels//,/ }; do echo --label "$l"; done)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)